stdlib-integration-tests/grpc_bidirectional_streaming/tests/grpc_bidirectional_streaming_pb.bal

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Though this is shown as this is not a new file. I moved this file.